packages/sdk/src/index.browser.ts

```diff
@@ -29,3 +29,20 @@ export {
 
 export { TeamsFx } from "./core/teamsfx.browser";
 export { IdentityType } from "./models/identityType";
+
+export {
+  NotificationTarget,
+  NotificationTargetType,
+  TeamsFxBotCommandHandler,
+} from "./conversation/interface.browser";
```

Comment:
It looks like the `interface.browser.ts` is same as `interface.ts`. If so, you could just export the latter one.